vms/platformvm/state/state.go

```diff
@@ -743,6 +831,83 @@ func (s *state) DeleteExpiry(entry ExpiryEntry) {
 	s.expiryDiff.DeleteExpiry(entry)
 }
 
+func (s *state) GetActiveSubnetOnlyValidatorsIterator() (iterator.Iterator[SubnetOnlyValidator], error) {
+	return s.sovDiff.getActiveSubnetOnlyValidatorsIterator(
+		s.activeSOVs.newIterator(),
+	), nil
+}
+
+func (s *state) NumActiveSubnetOnlyValidators() int {
+	return s.activeSOVs.len() + s.sovDiff.numAddedActive
```

Comment:
Nit: Would `numAddedActive` make more sense to name `numNetAddedActive` or `netAddedActive`? Implementation looks correct, but wasn't sure based on the name only

Reply:
Yeah, `net` seems like a good indicator of a possibly negative value.